packages/@aws-cdk/aws-ec2/lib/vpn.ts

```diff
@@ -75,6 +92,21 @@ export interface VpnConnectionOptions {
   readonly tunnelOptions?: VpnTunnelOption[];
 }
 
+/**
+ * The VpnGateway Properties
+ */
+export interface VpnGatewayProps extends CfnVPNGatewayProps {
+
+  /**
+   * Where to propagate VPN routes.
+   *
+   * @default - On the route tables associated with private subnets. If no
+   * private subnets exists, isolated subnets are used. If no isolated subnets
+   * exists, public subnets are used.
+   */
+  readonly vpnRoutePropagation?: SubnetSelection[]
```

Comment:
This property goes unused in the `VpnGateway` constructor.

Reply:
Not true. It is being used inside the function enableVpnGateway()

Reply:
I know, but traditionally `VpnGatewayProps` are used as the inputs to the `VpnGateway` class, and it's not the `VpnGateway` class that uses the `vpnRoutePropagation` field, it's `Vpc.enableVpnGateway`.

Here's how you probably want to set that up:

```ts
interface VpnGatewayProps {
  type, amazonSideAsn
}

interface EnableVpnGatewayOptions extends VpnGatewayProps {
  vpnRoutePropagation
}

class Vpc {
  public enableVpnGateway(options: EnableVpnGatewayOptions) { ... } 
}
```

Reply:
Thanks!!. Done